ansible/roles/opentelemetry/defaults/main.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 opentelemetry_namespace: opentelemetry-operator-system
 
-opentelemetry_version: v0.92.0
+opentelemetry_version: v0.114.1
```

Comment:
does this actually work in the cluster? I found I had to change the config for otel >= v0.104 because some {receivers, processors, exporters} were deprecated (https://github.com/SeldonIO/gcp-infra-automation-prototype/blob/main/tracing/k8s/otel-collector-beta1.yaml). Furthermore, the OpenTelemetryCollector CR moved from `opentelemetry.io/v1alpha1` to `opentelemetry.io/v1beta1` and for me that looked like this: https://github.com/SeldonIO/gcp-infra-automation-prototype/blob/main/tracing/k8s/otel-collector-beta1.yaml

Reply:
However, in that project I'm installing the operator via its helm chart. I've been using it as an experiment in updating dependencies, and I know we'll want to unify the ansible roles down the line.

Reply:
It does work but maybe we should move to the new api?

Reply:
![image](https://github.com/user-attachments/assets/dc577a00-92be-436e-bb29-283bff1dbd97)

Reply:
Perhaps what I saw was just some log spam as a result of some deprecated settings, I may have had a PR updating them for the main repo as well though